WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupViewModel.swift

```diff
@@ -184,9 +193,10 @@ private extension LoginJetpackSetupViewModel {
             guard let self else { return }
             switch result {
             case .success:
+                self.analytics.track(.loginJetpackSetupActivationSuccessful)
                 self.fetchJetpackConnectionURL()
             case .failure(let error):
-                // TODO: add tracks
+                self.analytics.track(.loginJetpackSetupActivationFailed)
```

Comment:
Similar as above:
```suggestion
                self.analytics.track(.loginJetpackSetupActivationFailed, withError: error)
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8187/commits/e6d2b8e1f7322fc2135566162fb5bb1a6e4b459c